tests/config/test_n2v_preprocessing.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+
+from careamics.config import N2VAlgorithm
+from careamics.config.support import (
+    SupportedStructAxis,
+    SupportedTransform,
+)
+from careamics.transforms import get_all_transforms
+
+# TODO name is confusing, and the tests are probably not in the right place. They should
```

Comment:
I agree with my own TODO comment. These tests don't belong here and might not be needed.